e2e/tests/implementation.ts

```diff
@@ -31,4 +33,10 @@ class Implementation {
       assert.equal(Implementation.vowelsCount(word), Number.parseInt(count));
     }
   }
+  @Step("This step uses a custom parameter of type Person and value <person>")
+  public async validatePerson(person: Person) {
+    assert.equal(person.name, "John");
+    assert.equal(person.age, 30);
```

Comment:
Can we change these to use `strictEqual`? I noticed that even if the parameter was declared as a `number` in earlier versions the number is actually passed as `string` and fails when we do a `===`

Reply:
Yes, this should be fix now, as we are doing primitive conversion. Just to highlight, parameters which are part of a table (string or int or bool) are still treated as `string`. As of now we are doing nested conversion.